### ISSUE TITLE
Errors/Warnings are not displayed in my header files

### ISSUE THREAD

**Issue by aIRisius** [hebaishi/linter-gcc#178]
## Checklist
* [x] Check that you have the GNU C Compiler installed by typing ```gcc --version``` into a terminal.
* [x] If you don't have the ```linter``` package installed, either update to the latest version of ```linter-gcc``` or install linter directly with ```apm install linter```.
* [x] Ensure that you are using a valid grammar - either C/C++/C++14
* [x] Ensure that you have read all the relevant information in the [wiki](https://github.com/hebaishi/linter-gcc/wiki), particularly the [Common Issues](https://github.com/hebaishi/linter-gcc/wiki/Common-Issues) page.

## Issue (including debugging messages):
Linter-gcc version: 0.7.1
Debug messages:
linter-gcc: Using configuration page settings

linter-gcc config: {"execPath":"/usr/bin/g++","gccIncludePaths":"./include","gccSuppressWarnings":false,"gccDefaultCFlags":"-c -Wall -Wextra -pedantic -std=c11","gccDefaultCppFlags":"-c -Wall -Wextra -pedantic -std=c++17","gccErrorLimit":-1,"gccErrorString":"error","gccWarningString":"warning","gccNoteString":"note","compileCommandsFile":"./build/compile_commands.json"}

/home/maxim/jaguar/build/compile_commands.json

linter-gcc: /usr/bin/g++ -c -Wall -Wextra -pedantic -std=c++17 -I/home/maxim/jaguar/include /home/maxim/jaguar/include/expression

When I was using sublime text, I created header files with no extension, but with first line
```c++
// -*- C++ -*-
```
and syntax hightlights worked perfectly.

When I migrated to Atom, installed all the packages, including linter-gcc,
I noticed that warnings and errors are not displayed anywhere, when I was editing these header files,
but in my source files warnings and errors are shown.

P.S Sorry for my bad english.
